libcaf_core/caf/json_builder.test.cpp

```diff
@@ -148,7 +148,7 @@ TEST("float") {
     check(builder.value(4.2f));
```

Comment:
This will widen the `float` to a `double` anyways. Let's just remove this `SECTION`. It's redundant with `TEST("double")` below.